packml_stats_loader/include/packml_stats_loader/packml_stats_loader.h

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2019, PlusOne Robotics
+ * All rights reserved.
+*/
+
+#ifndef SRC_PACKML_STATS_LOADER_H
+#define SRC_PACKML_STATS_LOADER_H
+
+#include <ros/ros.h>
+#include <rosbag/bag.h>
+#include <rosbag/view.h>
+#include <packml_msgs/GetStats.h>
+#include <packml_msgs/LoadStats.h>
+#include <packml_msgs/Stats.h>
+
+namespace packml_stats_loader
+{
+
+  class PackmlStatsLoader
+  {
+  public:
+    /**
+     * @brief Constructor
+     * @param pnh Private node handle
+     */
+    PackmlStatsLoader(const ros::NodeHandle& pnh);
+
+  private:
+    packml_msgs::Stats loadStats();
+    void writeStats(const packml_msgs::GetStats::Response& get_stats_response);
```

Comment:
Not a big deal, but stats "loader" also handles writing?

Also, it seems like it might be advantageous to call these externally, for instance before shutting down. Did you consider making these functions public?

Reply:
Yeah, I didn't want to make the name too long, so I just called it loader (though in hindsight StatsUtil probably would have been better).

I was thinking of this just as a one-off node, but that makes a lot of sense, changed to public.